src/simplify_algebra.cpp

```diff
@@ -185,6 +187,41 @@ struct find_mul_add
     }
 };
 
+// find gelu (a multi-step formula), using the tanh implementation. Replace it with the gelu_erf
+// formula
+struct find_gelu
+{
+    // compile option fast_math controls whether to use math shortcuts like this
+    bool fast_math = true;
+
+    auto matcher() const { return match::gelu_tanh(); }
+
+    // apply the gelu_erf formula: return x * 0.5 * (1 + ::erf(x * M_SQRT1_2));
+    //  = x * (0.5 + 0.5 * ::erf(x * M_SQRT1_2)) after match/transform
+    void apply(module& p, match::matcher_result r) const
```

Comment:
This should be `module& m` instead of `module& p`.

Reply:
changed